workshop/geopython-workshop-ctl.ps.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+PROGRAM_NAME=$(basename $0)
+
+USAGE="Usage: $PROGRAM_NAME <start|stop|url|update|clean>"
+
+if [ "$#" -ne 1 ]; then
+    echo $USAGE
+    exit 1
+fi
+
+# Test for the command
+if [ $1 == "start" ]; then
+    $0 stop
+    docker compose up -d
+elif [ $1 == "stop" ]; then
+    docker compose stop
+    docker compose rm --force
+elif [ $1 == "url" ]; then
+    # try to open the Jupyter Notebook in Browser
+    platform="$(uname | tr '[:upper:]' '[:lower:]')"
```

Comment:
is this still required? I guess the `openapp="cmd /c start"` is the command in PS here.